tests/h/models/user_test.py

```diff
@@ -289,6 +289,19 @@ def test_it_filters_by_authority(self, db_session, users):
         actual = models.User.get_by_username(db_session, user.username, 'example.com')
         assert actual is None
 
+    def test_annotation_count_returns_count_of_shared_annotations_for_user(self, db_session, users):
+        user = users['meredith']
+
+        for i in range(50):
+            annotation(db_session,
+                       document(db_session,
+                                'http://example.com/document_' + str(i)),
+                       user.userid,
+                       shared=True)
```

Comment:
I think you can just use the factories here, which saves you quite some typing and will be more future proof when we change the annotation/document models.
Which means that you won't need the annotation and document fixture but can just do `factories.Annotation(user=user.userid, shared=True)` in the test above where you use it.